app/controllers/planning_application/site_notices_controller.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+class PlanningApplication
+  class SiteNoticesController < AuthenticationController
+    include ActionView::Helpers::SanitizeHelper
+
+    before_action :set_planning_application
+    before_action :set_site_notice, except: %i[new create]
+    before_action :ensure_public_portal_is_active, only: :create
+
+    def show
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def new
+      @site_notice = SiteNotice.new
+    end
+
+    def edit
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def create
+      @site_notice = SiteNotice.new(site_notice_params
+                                    .except(:method, :internal_team_email)
+                                    .merge(planning_application_id: @planning_application.id))
+
+      if @site_notice.save
+        @site_notice.update(content: @site_notice.preview_content(@planning_application))
+
+        if params[:commit] == "Email site notice and mark as complete"
+          if site_notice_params[:internal_team_email].presence
```

Comment:
Is this something they need to fill out each time or would it be consistent enough to add a field on the local_authority table?